src/site/includes/veteran-navigation.html

```diff
@@ -1,165 +0,0 @@
-                  <!-- submenu trigger -->
-                  <li>
-
-                    <button
-                    class="vetnav-level2 back-button"
-                    aria-controls="vetnav-pension">Back to Menu</button>
-                    <div class="vetnav-level2-heading">Pension</div>
-                  <li><a href="/pension/" onClick="reportHeaderNav('Explore Benefits->Pension');">Pension Benefits Overview</a></li>
-                  <li><a href="/pension/eligibility/" onClick="reportHeaderNav('Explore Benefits->Pension');">Eligibility</a></li>
-                  <li><a href="/pension/how-to-apply/" onClick="reportHeaderNav('Explore Benefits->Pension');">How to Apply for a Veterans Pension</a></li>
-                  <li><a href="/pension/survivors-pension/" onClick="reportHeaderNav('Explore Benefits->Pension');">How to Apply for a Survivors Pension</a></li>
-                </ul>
-                <!-- end submenu panel -->
-              </li>
-              <!-- end menu item -->
-
-              <li>
-                <button class="vetnav-level2 vetnav-trigger" role="button" aria-haspopup="true" aria-controls="vetnav-budget-performance" onClick="recordEvent({ event: 'nav-header-second-level' });">Budget and Performance</button>
-
-                <ul class="vetnav-panel vetnav-panel--submenu" id="vetnav-about-va" role="menu" aria-label="Budget and Performance" hidden>
-                  <!-- submenu trigger -->
-                  <li>
-                    <button class="vetnav-back vetnav-trigger" aria-controls="vetnav-about-va">Back to menu</button>
-                    <div class="vetnav-level2-heading">Budget and Performance</div>
-                  </li>
-                  <!-- submenu links-->
-                  <li><a href="https://www.va.gov/performance/">VA Plans, Budget, and Performance</a></li>
-                  <li><a href="https://www.innovation.va.gov/">VA Center for Innovation (VACI)</a></li>
-                  <li><a href="https://www.va.gov/budget/report/">Agency Financial Report (AFR)</a></li>
-                  <li><a href="https://www.va.gov/budget/products.asp">Budget Submission</a></li>
-                  <li><a href="https://www.va.gov/recovery/">Recovery Act</a></li>
-                  <!-- end submenu links -->
-                </ul>
-                <!-- end submenu panel -->
-              </li>
-              <!-- end second menu item -->
-            </ul>
-          </div>
-        </li>
-
-        <!-- LOCATIONS -->
-        <li><a href="/facilities/" class="vetnav-level1" id="fxkfp">Find VA Locations<script nonce="">(function() { var e = document.getElementById('fxkfp'); e.addEventListener('click', function(ev) { reportHeaderNav('Find VA locations'); }); })();</script></a></li>
-
-      </ul>
-      <!-- end global nav ul -->
-    </nav>
-  </div>
-</div>
-
-<script>
-function reportHeaderNav(navpath) {
-  recordEvent({
-    event: 'nav-header-link',
-    'nav-path': navpath
-  });
-}
-</script>
```

Comment:
@cvalarida I've never been clear, but with this being removed, do we still need `src/platform/site-wide/accessible-menus.js`? I thought that wasn't ncessary anymore since the menu moved to React.

Reply:
A couple e2e tests too, `src/platform/site-wide/tests/accessible-menus.e2e.spec.js`